recipes/genometools-genometools/build.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
-make
+
+
+make cairo=no
```

Comment:
Any reason to disable it? I think we have all cairo dependencies in conda-forge.